Windows system allows headers to be case insensitive but during cross-compile
MinGW requires these to be lowecased.

find . -type f -name 'CMakeCache.txt' -exec rm -v {} +
grep -rln '#include <Windows.h>' | xargs sed -i 's/Windows.h/windows.h/g'
grep -rln '#include <ShlObj.h>' | xargs sed -i 's/ShlObj.h/shlobj.h/'
grep -rln '#include <VersionHelpers.h>' | xargs sed -i 's/VersionHelpers.h/versionhelpers.h/'
grep -rln '#include <Commctrl.h>' | xargs sed -i 's/Commctrl.h/commctrl.h/'
grep -rln '#include <Richedit.h>' | xargs sed -i 's/Richedit.h/richedit.h/'
grep -rln '#include <WinInet.h>' | xargs sed -i 's/WinInet.h/wininet.h/'

Signed-off-by: Leela Venkaiah G <leelavg@thoughtexpo.com>

diff --git a/src/draw2d/win/draw2d_gdi.ixx b/src/draw2d/win/draw2d_gdi.ixx
index e15f16e..294e458 100644
--- a/src/draw2d/win/draw2d_gdi.ixx
+++ b/src/draw2d/win/draw2d_gdi.ixx
@@ -15,7 +15,7 @@
 
 #include <draw2d/draw2d.hxx>
 #include <sewer/nowarn.hxx>
-#include <Windows.h>
+#include <windows.h>
 #include <sewer/warn.hxx>
 
 struct _measurestr_t
diff --git a/src/inet/win/oshttpreq.c b/src/inet/win/oshttpreq.c
index 5a76201..fb8ff63 100644
--- a/src/inet/win/oshttpreq.c
+++ b/src/inet/win/oshttpreq.c
@@ -25,8 +25,8 @@
 #endif
 
 #include <sewer/nowarn.hxx>
-#include <Windows.h>
-#include <WinInet.h>
+#include <windows.h>
+#include <wininet.h>
 #include <sewer/warn.hxx>
 
 struct _oshttp_t
diff --git a/src/osapp/osmain_win.h b/src/osapp/osmain_win.h
index 5daac22..ffb2dd5 100644
--- a/src/osapp/osmain_win.h
+++ b/src/osapp/osmain_win.h
@@ -11,7 +11,7 @@
 /* Cross-platform main */
 
 #include <sewer/nowarn.hxx>
-#include <Windows.h>
+#include <windows.h>
 #include <sewer/warn.hxx>
 
 #if (_MSC_VER)
diff --git a/src/osapp/win/osapp_win.c b/src/osapp/win/osapp_win.c
index 3104edf..b7ebae3 100644
--- a/src/osapp/win/osapp_win.c
+++ b/src/osapp/win/osapp_win.c
@@ -23,8 +23,8 @@
 #endif
 
 #include <sewer/nowarn.hxx>
-#include <Windows.h>
-#include <ShlObj.h>
+#include <windows.h>
+#include <shlobj.h>
 #include <sewer/warn.hxx>
 
 struct _osapp_t
diff --git a/src/osbs/win/bfile.c b/src/osbs/win/bfile.c
index c26de37..e032e6e 100644
--- a/src/osbs/win/bfile.c
+++ b/src/osbs/win/bfile.c
@@ -22,8 +22,8 @@
 #endif
 
 #include <sewer/nowarn.hxx>
-#include <Windows.h>
-#include <ShlObj.h>
+#include <windows.h>
+#include <shlobj.h>
 #include <sewer/warn.hxx>
 
 struct _dir_t
diff --git a/src/osbs/win/bmutex.c b/src/osbs/win/bmutex.c
index 1e08310..1f45f21 100644
--- a/src/osbs/win/bmutex.c
+++ b/src/osbs/win/bmutex.c
@@ -19,7 +19,7 @@
 #endif
 
 #include <sewer/nowarn.hxx>
-#include <Windows.h>
+#include <windows.h>
 #include <sewer/warn.hxx>
 
 /*---------------------------------------------------------------------------*/
diff --git a/src/osbs/win/bproc.c b/src/osbs/win/bproc.c
index cf492ca..c1d85b5 100644
--- a/src/osbs/win/bproc.c
+++ b/src/osbs/win/bproc.c
@@ -22,7 +22,7 @@
 #endif
 
 #include <sewer/nowarn.hxx>
-#include <Windows.h>
+#include <windows.h>
 #include <sewer/warn.hxx>
 
 #define STDIN_READ_CHILD 0
diff --git a/src/osbs/win/bthread.c b/src/osbs/win/bthread.c
index eb3ac80..634065a 100644
--- a/src/osbs/win/bthread.c
+++ b/src/osbs/win/bthread.c
@@ -19,7 +19,7 @@
 #endif
 
 #include <sewer/nowarn.hxx>
-#include <Windows.h>
+#include <windows.h>
 #include <sewer/warn.hxx>
 
 /*---------------------------------------------------------------------------*/
diff --git a/src/osbs/win/btime.c b/src/osbs/win/btime.c
index e95ac99..88e3d5e 100644
--- a/src/osbs/win/btime.c
+++ b/src/osbs/win/btime.c
@@ -18,7 +18,7 @@
 #endif
 
 #include <sewer/nowarn.hxx>
-#include <Windows.h>
+#include <windows.h>
 #include <sewer/warn.hxx>
 
 #define EPOCHFILETIME (116444736000000000LL)
diff --git a/src/osbs/win/dlib.c b/src/osbs/win/dlib.c
index a70e30c..c6e398e 100644
--- a/src/osbs/win/dlib.c
+++ b/src/osbs/win/dlib.c
@@ -22,7 +22,7 @@
 #endif
 
 #include <sewer/nowarn.hxx>
-#include <Windows.h>
+#include <windows.h>
 #include <sewer/warn.hxx>
 
 /*---------------------------------------------------------------------------*/
diff --git a/src/osbs/win/sinfo.c b/src/osbs/win/sinfo.c
index eb8cf7c..ac25f61 100644
--- a/src/osbs/win/sinfo.c
+++ b/src/osbs/win/sinfo.c
@@ -20,7 +20,7 @@
 
 #include <sewer/nowarn.hxx>
 
-#include <Windows.h>
+#include <windows.h>
 #undef __VERSION_HELPERS__
 
 #if defined(_MSC_VER)
@@ -32,7 +32,7 @@
 #endif
 
 #if defined __VERSION_HELPERS__
-#include <VersionHelpers.h>
+#include <versionhelpers.h>
 #endif
 
 #include <sewer/warn.hxx>
diff --git a/src/osgui/win/oscomwin.c b/src/osgui/win/oscomwin.c
index 19527fd..54c5384 100644
--- a/src/osgui/win/oscomwin.c
+++ b/src/osgui/win/oscomwin.c
@@ -23,8 +23,8 @@
 #endif
 
 #include <sewer/nowarn.hxx>
-#include <Windows.h>
-#include <ShlObj.h>
+#include <windows.h>
+#include <shlobj.h>
 #include <sewer/warn.hxx>
 
 /*---------------------------------------------------------------------------*/
diff --git a/src/osgui/win/osgui_win.cpp b/src/osgui/win/osgui_win.cpp
index bf63984..c95c869 100644
--- a/src/osgui/win/osgui_win.cpp
+++ b/src/osgui/win/osgui_win.cpp
@@ -27,8 +27,8 @@
 #endif
 
 #include <sewer/nowarn.hxx>
-#include <Commctrl.h>
-#include <Richedit.h>
+#include <commctrl.h>
+#include <richedit.h>
 #include <uxtheme.h>
 #include <gdiplus.h>
 #include <sewer/warn.hxx>
diff --git a/src/osgui/win/osimglist.c b/src/osgui/win/osimglist.c
index 29dcd72..e9c5ddc 100644
--- a/src/osgui/win/osimglist.c
+++ b/src/osgui/win/osimglist.c
@@ -20,7 +20,7 @@
 #include <sewer/ptr.h>
 
 #include <sewer/nowarn.hxx>
-#include <Commctrl.h>
+#include <commctrl.h>
 #include <sewer/warn.hxx>
 
 struct _osimglist_t
diff --git a/src/osgui/win/osprogress.c b/src/osgui/win/osprogress.c
index ab554b5..05ce058 100644
--- a/src/osgui/win/osprogress.c
+++ b/src/osgui/win/osprogress.c
@@ -22,7 +22,7 @@
 #endif
 
 #include <sewer/nowarn.hxx>
-#include <Commctrl.h>
+#include <commctrl.h>
 #include <sewer/warn.hxx>
 
 struct _osprogress_t
diff --git a/src/osgui/win/osslider.c b/src/osgui/win/osslider.c
index a211714..4b70037 100644
--- a/src/osgui/win/osslider.c
+++ b/src/osgui/win/osslider.c
@@ -26,7 +26,7 @@
 #endif
 
 #include <sewer/nowarn.hxx>
-#include <Commctrl.h>
+#include <commctrl.h>
 #include <sewer/warn.hxx>
 
 struct _osslider_t
diff --git a/src/osgui/win/ostext.c b/src/osgui/win/ostext.c
index 2c80255..7ddf3de 100644
--- a/src/osgui/win/ostext.c
+++ b/src/osgui/win/ostext.c
@@ -31,7 +31,7 @@
 #endif
 
 #include <sewer/nowarn.hxx>
-#include <Richedit.h>
+#include <richedit.h>
 #include <sewer/warn.hxx>
 
 struct _ostext_t
diff --git a/src/osgui/win/ostooltip.c b/src/osgui/win/ostooltip.c
index d3ce605..f610497 100644
--- a/src/osgui/win/ostooltip.c
+++ b/src/osgui/win/ostooltip.c
@@ -20,7 +20,7 @@
 #endif
 
 #include <sewer/nowarn.hxx>
-#include <Commctrl.h>
+#include <commctrl.h>
 #include <sewer/warn.hxx>
 
 /*---------------------------------------------------------------------------*/
diff --git a/src/osgui/win/osupdown.c b/src/osgui/win/osupdown.c
index 22d3fe8..664cce9 100644
--- a/src/osgui/win/osupdown.c
+++ b/src/osgui/win/osupdown.c
@@ -26,7 +26,7 @@
 #endif
 
 #include <sewer/nowarn.hxx>
-#include <Commctrl.h>
+#include <commctrl.h>
 #include <sewer/warn.hxx>
 
 struct _osupdown_t
diff --git a/src/osgui/win/osview.cpp b/src/osgui/win/osview.cpp
index f43c5e9..40d16c9 100644
--- a/src/osgui/win/osview.cpp
+++ b/src/osgui/win/osview.cpp
@@ -36,7 +36,7 @@
 
 #include <sewer/nowarn.hxx>
 #include <gdiplus.h>
-#include <Commctrl.h>
+#include <commctrl.h>
 #include <sewer/warn.hxx>
 
 struct _osview_t
diff --git a/src/sewer/win/bstdimp.c b/src/sewer/win/bstdimp.c
index e3cda3d..08bef97 100644
--- a/src/sewer/win/bstdimp.c
+++ b/src/sewer/win/bstdimp.c
@@ -22,7 +22,7 @@
 #endif
 
 #include "nowarn.hxx"
-#include <Windows.h>
+#include <windows.h>
 #include <stdio.h>
 #include "warn.hxx"
 
